src/xiaomi_ble/devices.py

```diff
@@ -226,3 +226,6 @@ class DeviceEntry:
         model="SU001-T",
     ),
 }
+
+
+SLEEPY_DEVICE_MODELS = ["CGH1", "MCCGQ02HL"]
```

Comment:
Should we make this a set to avoid linear searches in the future if it gets big?

Reply:
Fixed